### ISSUE TITLE
How to connect to custom repository that requires authentication

### ISSUE THREAD

**Issue by github-ronk** [bestwu/gradle-dependencies-plugins-helper-plugin#7]
We have a custom repository (Artifactory) which is used in gradle builds,

Successfully configured to use it, using the URL field of the "Nexus2 Repository Search"

However, the plugin cannot connect to it, because this repo requires authentication (user and password)

And it seems that authentication is not supported by the plugin (?)

How can this be done?

Thanks!

**Comment by github-ronk** [comment]
To clarify, the response shown in the IntelliJ Event Log is:

```
12:44	No dependencies found
			response: {
				"errors" : [ {
				"status" : 401,
				"message" : "Authentication is required"
				} ]
			}.

```

**Comment by github-ronk** [comment]
@bestwu , many thanks for the fix.  I see it's merged in release 0.1.7, could you please make that version available for installation in IntelliJ?  Thanks 🙂
